### ISSUE TITLE
Keyboard focus is moving to the links under menu button when the menu button is collapsed in Windows.

### ISSUE THREAD

**Issue by msftedad** [docsifyjs/docsify#2605]
### Description

**User Impact:**
Keyboard-dependent users may face confusion when focus shifts to hidden links after collapsing the menu. This unexpected behavior disrupts navigation flow, increases the risk of accidental actions, and negatively impacts accessibility.

**Test Environment:** 
OS: Windows 11 Enterprise Insider Preview Version 24H2 (OS Build 26100.2894)
Browser Microsoft Edge Version 138.0.3351.55
Application URL: https://docsify.js.org/#/more-pages?id=sidebar

**Refer Attachment:** 
1.Keyboard focus is moving to the links under menu button when the menu button is collapsed in Windows.png
2.Keyboard focus is moving to the links under menu button when the menu button is collapsed in Windows.mp4

https://github.com/user-attachments/assets/117c427a-bf91-45c3-aa30-0a86aaf85813
<img width="960" height="549" alt="Image" src="https://github.com/user-attachments/assets/2523e983-fc0e-4ea2-82ac-8d701cd1bf75" />

### Expected behavior

On pressing tab key, focus shouldn't move to the links present in the collapsed menu in Windows.

### Actual behavior

On pressing tab key after menu button, focus moves to the links present inside the menu button even when the menu is in collapsed state in Windows.

### Steps to reproduce

1. Open the URL: https://docsify.js.org/#/more-pages?id=sidebar in latest edge browser.
2. Augmentation loop offerings page will appear.
3. Press tab key to move to the 'menu' control.
4. Verify whether keyboard focus is moving to the links under menu button when the menu button is collapsed or not.

### Environment

OS: Windows 11 Enterprise Insider Preview Version 24H2 (OS Build 26100.2894)
Browser Microsoft Edge Version 138.0.3351.55
Application URL: https://docsify.js.org/#/more-pages?id=sidebar

### Additional Information

- [x] Bug still occurs when all/other plugins are disabled?

**Comment by sy-records** [comment]
I cannot access web.augloop-tools.officeppe.com

Please ensure testing is conducted based on v5.0.0-rc3.
